[2025-08-07 03:47:25,001] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-07T03:47:23.590669+00:00 [queued]>
[2025-08-07 03:47:25,021] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-07T03:47:23.590669+00:00 [queued]>
[2025-08-07 03:47:25,021] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 03:47:25,021] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-07 03:47:25,021] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 03:47:25,051] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): fetch_and_send_weather> on 2025-08-07 03:47:23.590669+00:00
[2025-08-07 03:47:25,062] {standard_task_runner.py:52} INFO - Started process 4245 to run task
[2025-08-07 03:47:25,071] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'fetch_and_send_weather', 'manual__2025-08-07T03:47:23.590669+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpm79rmvmh', '--error-file', '/tmp/tmp9e28ioa1']
[2025-08-07 03:47:25,073] {standard_task_runner.py:80} INFO - Job 107: Subtask fetch_and_send_weather
[2025-08-07 03:47:25,210] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-07T03:47:23.590669+00:00 [running]> on host 92a46ac2453f
[2025-08-07 03:47:25,546] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alla
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=fetch_and_send_weather
AIRFLOW_CTX_EXECUTION_DATE=2025-08-07T03:47:23.590669+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-07T03:47:23.590669+00:00
[2025-08-07 03:47:25,547] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 03:47:25,548] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/scripts && python3 kafka/producer.py']
[2025-08-07 03:47:25,575] {subprocess.py:85} INFO - Output:
[2025-08-07 03:47:27,391] {subprocess.py:92} INFO - Envoi vers Kafka : {'temperature': 28, 'humidity': 84, 'windSpeed': 3.1, 'cloudCover': 23, 'rainIntensity': 0, 'timestamp': '2025-08-07T03:47:00Z'}
[2025-08-07 03:47:27,507] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 03:47:27,585] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=fetch_and_send_weather, execution_date=20250807T034723, start_date=20250807T034725, end_date=20250807T034727
[2025-08-07 03:47:27,649] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-07 03:47:27,735] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
